routes/users.py

```diff
@@ -393,3 +393,31 @@ def suggest():
     result["users"]=[_[0] for _ in heapq.nlargest(min(10,len(list_of_users)),list_of_users,key=lambda x: x[1])]
     
     return result
+
+
+
+@app.route("/users/warn", methods=["POST"])
```

Comment:
Approving a report automatically sends a warning. There is no need for this endpoint.